provisioners.proto

```diff
@@ -166,13 +167,21 @@ message K8sSAProvisioner {
 
 message SSHPOPProvisioner {}
 
+message SCEPDecrypter {
+	config.KMS kms = 1;
+	bytes decrypterCertificate = 2;
+	string decrypterKey = 3;
```

Comment:
Both should be the same type, but we have mixed values between the certificates and keys in the Configuration type, and the ones in the provisioners.

Reply:
To verify: `decrypterKey` should thus become `bytes` too? Then try base64 decoding to see if there's a PEM formatted key in there, and if so, use that as the (softkms) key? If base64 decoding fails, try interpreting it as a key URI?

Reply:
In my opinion, because the provisioners can be configured remotely, we should send store the key itself, a PEM, or the URI that represents it. For a PEM, bytes are "easier"; for the URI, a string is easier. We can consider having two fields if it makes this easier. We can use oneof or Any, but these are more difficult than just adding two incompatible fields that will be verified on the server side.

Reply:
This change is in f2f59aa4249bc91540ae15617cc84ee1bde5b308.